### PR TITLE
Parameter sweeping for similarity

### PR DESCRIPTION
**Pull request by PatrickBue**
### Description
Slightly modified parameter sweeping to work for image similarity.

### Checklist:
- [x] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [x] I have added tests covering my contributions.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by PatrickBue** [comment]
Will add unit- and integration-tests in next commit

File: utils_cv/classification/parameter_sweeper.py [hunk 1]
```diff
@@ -440,6 +446,7 @@ def run(
 
         res = dict()
         for rep in range(reps):
+            print
```
Comment (jiata) [inline utils_cv/classification/parameter_sweeper.py]: extra print line

File: tests/integration/similarity/test_integration_similarity_notebooks.py [hunk 2]
```diff
@@ -26,3 +25,17 @@ def test_01_notebook_run(similarity_notebooks):
...
+def test_11_notebook_run(similarity_notebooks, tiny_ic_data_path):
+    if linux_with_gpu():
+        notebook_path = similarity_notebooks["11"]
+        pm.execute_notebook(
+            notebook_path,
+            OUTPUT_NOTEBOOK,
+            parameters=dict(PM_VERSION=pm.__version__),
+            kernel_name=KERNEL_NAME,
+        )
+        # nb_output = sb.read_notebook(OUTPUT_NOTEBOOK)
```
Comment (jiata) [inline tests/integration/similarity/test_integration_similarity_notebooks.py]: remove commented lines (if all we're testing for is that it runs)

Comment (PatrickBue) [reply tests/integration/similarity/test_integration_similarity_notebooks.py]: Done

File: utils_cv/classification/parameter_sweeper.py (hunk 1 above)
Comment (PatrickBue) [reply utils_cv/classification/parameter_sweeper.py]: done

**Pull request merged**